public class org.apache.flink.testutils.Driver extends java.lang.Object
{

    public void <init>()
    {
        org.apache.flink.testutils.Driver r0;

        r0 := @this: org.apache.flink.testutils.Driver;

        specialinvoke r0.<java.lang.Object: void <init>()>();

        return;
    }

    public void runall()
    {
        org.apache.flink.testutils.Driver r0;
        org.apache.flink.testutils.ClassLoaderUtilsTest $r2;
        java.lang.Exception $r3, $r4, $r5, $r6;

        r0 := @this: org.apache.flink.testutils.Driver;

        $r2 = new org.apache.flink.testutils.ClassLoaderUtilsTest;

        specialinvoke $r2.<org.apache.flink.testutils.ClassLoaderUtilsTest: void <init>()>();

     label01:
        virtualinvoke $r2.<org.apache.flink.testutils.ClassLoaderUtilsTest: void testObjectFromNewClassLoaderObject()>();

     label02:
        goto label04;

     label03:
        $r4 := @caughtexception;

        virtualinvoke $r4.<java.lang.Exception: void printStackTrace()>();

     label04:
        virtualinvoke $r2.<org.apache.flink.testutils.ClassLoaderUtilsTest: void testObjectFromNewClassLoaderClassLoaders()>();

     label05:
        goto label07;

     label06:
        $r5 := @caughtexception;

        virtualinvoke $r5.<java.lang.Exception: void printStackTrace()>();

     label07:
        virtualinvoke $r2.<org.apache.flink.testutils.ClassLoaderUtilsTest: void testExceptionObjectFromNewClassLoaderObject()>();

     label08:
        goto label10;

     label09:
        $r3 := @caughtexception;

        virtualinvoke $r3.<java.lang.Exception: void printStackTrace()>();

     label10:
        virtualinvoke $r2.<org.apache.flink.testutils.ClassLoaderUtilsTest: void testExceptionObjectFromNewClassLoaderClassLoaders()>();

     label11:
        goto label13;

     label12:
        $r6 := @caughtexception;

        virtualinvoke $r6.<java.lang.Exception: void printStackTrace()>();

     label13:
        return;

        catch java.lang.Exception from label01 to label02 with label03;
        catch java.lang.Exception from label04 to label05 with label06;
        catch java.lang.Exception from label07 to label08 with label09;
        catch java.lang.Exception from label10 to label11 with label12;
    }
}
